mitodl/ocw-hugo-themes
# Update OCW Staff Team Listing

# What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/2520

# Description (What does it do?)
Updates OCW Staff Team listing


# How can this be tested?
1. Checkout to this branch
2. `yarn start www`
3. Go to this page and confirm the staff listing requirements from the ticket - http://localhost:3000/about/
